float32[2] startPoint
float32[2] endPoint
---
float32[] R0Frames
float32[] RAFrames